agent_app_sample_code/utils/chunk_docs.py

```diff
@@ -0,0 +1,96 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC #### `chunk_docs`
+# MAGIC
+# MAGIC `chunk_docs` creates a new delta table, given a table of documents, computing the chunk function over each document to produce a chunked documents table. This utility will let you write the core business logic of the chunker, without dealing with the spark details. You can decide to write your own, or edit this code if it does not fit your use case.
```

Comment:
I wonder if we should just reuse the simpler chunking example from Databricks docs in https://docs.databricks.com/en/generative-ai/create-query-vector-search.html#notebook-examples-for-calling-an-embeddings-model?

Reply:
yea thats fair. i think the advantage of the recursive text splitter is that this would probably give u a higher baseline quality. 

this is one the primary levers for retrieval quality, so i think its good that we document some options on how you would iterate on this.